ZFSin/zfs/lib/libzfs/libzfs_mount.c

```diff
@@ -240,7 +240,7 @@ dir_is_empty_stat(const char *dirname)
 	 * We only want to return false if the given path is a non empty
 	 * directory, all other errors are handled elsewhere.
 	 */
-	if (stat(dirname, &st) < 0 || !S_ISDIR(st.st_mode)) {
+	if (stat(dirname, (struct stat*)&st) < 0 || !S_ISDIR(st.st_mode)) {
```

Comment:
Not sure we should define `_stat64` then typecast it, probably better to change the type to fit the call.